tests/baselines/reference/enumLiteralTypes3.errors.txt

```diff
@@ -31,7 +33,8 @@ tests/cases/conformance/types/literal/enumLiteralTypes3.ts(96,14): error TS2678:
 !!! error TS2322:   Type 'Choice.No' is not assignable to type 'Choice.Yes'.
         a = c;
         ~
-!!! error TS2322: Type 'Choice' is not assignable to type 'Choice.Yes'.
+!!! error TS2322: Type 'UnknownYesNo' is not assignable to type 'Choice.Yes'.
+!!! error TS2322:   Type 'Choice.Unknown' is not assignable to type 'Choice.Yes'.

```

Comment:
This one actually seems kind of unfortunate.

Reply:
Why? It's now elaborating on the reason.